subprojects/zydis.wrap

```diff
@@ -0,0 +1,6 @@
+[wrap-git]
+url = https://github.com/zyantific/zydis.git
+directory = zydis
+patch-directory = zydis
+depth = 1
+revision = master
```

Comment:
Is there any big change in the `master` branch? Otherwise sticking to a released package (and using `wrap-file`) is preferred.